Xamarin.iOS.sln

```diff
@@ -15,7 +15,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "link sdk", "tests\linker\io
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "dont link", "tests\linker\ios\dont link\dont link.csproj", "{839212D5-C25B-4284-AA96-59C3872B8184}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "mtouch", "tests\mtouch\mtouch.csproj", "{9A1177F5-16E6-45DE-AA69-DC9924EC39B8}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "mtouchtests", "tests\mtouch\mtouchtests.csproj", "{9A1177F5-16E6-45DE-AA69-DC9924EC39B8}"

```

Comment:
I've created a separate PR with my changes, to not complicate this one too much: https://github.com/xamarin/xamarin-macios/pull/11400

Reply:
Just to make sure I don't mess this up again haha are these the steps I should do?
1) land your PR first
2) 'git rm -cached' on the files that are in your PR
3) fetch and merge main in my branch and push to this PR?

Reply:
That's right! You probably don't need step #2, but it shouldn't hurt either.

Reply:
Thanks!